plugin/index.js

```diff
@@ -38,16 +38,16 @@ function observeBundleMetrics(stats, inspectpack) {
   const bundlesToObserve = Object.keys(stats.compilation.assets)
     .filter(
       bundlePath =>
-        // Don't include hot reload assets, they break everything
+        // Don"t include hot reload assets, they break everything
```

Comment:
Your linter/editor seems to be unnecessarily changing all the single quotes to double quotes, as well as removing dangling commas.

Reply:
Ah, I find-and-replaced too hard here. The reverse is true though: a previous commit must've formatted this with custom Prettier plugin settings or something. This corrects it to fit into our lint rules.